P1/style.css

```diff
@@ -0,0 +1,176 @@
+BODY {
+    background: linear-gradient(90deg, rgb(44,44,44) 50%, rgb(50,50,50) 50%,rgb(50,50,50) 55%, rgb(40,40,40) 55%);
+    background-size: 40px 40px;
+    background-color: black;
+    padding-top: 39px;
+    font-family: "Source Sans Pro";
+    /*letter-spacing: 1px;*/
+}
+
+div {
+    padding-top: 21px;
```

Comment:
bad practice to use a common style for `div` in such way. padding all the things :)